okteto.yml

```diff
@@ -0,0 +1,23 @@
+build:
+  go-guestbook:
+    image: okteto.dev/go-guestbook:1.0.0
+    context: .
+
+deploy:
+  - kubectl apply -k manifests
```

Comment:
let's keep `kustomize edit set image okteto/guestbook:golang=$OKTETO_BUILD_GUESTBOOK_IMAGE` and remove the image field from https://github.com/okteto/go-guestbook/pull/5/files#diff-dd8070292ce806a326a3d0543b136ba3b5f7930f40d33475780462fdcece2b70R3